README.md

```diff
@@ -1,213 +1,44 @@
+1. Alfresco Repository for Community, with:  
+1.1. Alfresco Share Services AMP  
+1.2. Alfresco AOS AMP  
+1.3. Alfresco vti-bin war - helps with AOS integration  
+1.4. Alfresco Google Docs Repo AMP  
+2. Alfresco Share, with:  
+2.1 Alfresco Google Docks Share AMP  
+3. A Postgres DB  
+4. Alfresco Search Services (Solr6)  
 
-#### Notes:  
-Not all combination of image tags may work, please use the recommended ones.
+## Deployment options
 
-#### Other information
+To deploy Alfresco Content Services Community, follow the steps in the [acs-deployment](https://github.com/Alfresco/acs-deployment) project using the links provided.
 
-You can find the helm chart [here](https://github.com/Alfresco/acs-community-deployment/tree/master/helm/alfresco-content-services-community).
+**Note:** You'll need to adapt some steps and commands for the Community release. Replace any occurrences of `alfresco-content-repository` with `alfresco-content-repository-community` to ensure you're using the Community release. Example commands may need to be changed when cloning the Community project, changing directory, or deploying the Community Helm chart:
+* [Deploying with Helm charts on AWS](https://github.com/Alfresco/acs-deployment/blob/master/docs/helm-deployment-aws_cloud.md)
+* [Deploying with Helm charts using Minikube](https://github.com/Alfresco/acs-deployment/blob/master/docs/helm-deployment-minikube.md)
+* [Deploying using Docker Compose](https://github.com/Alfresco/acs-deployment/blob/master/docs/docker-compose-deployment.md)
 
-The system should handle pod recreation without any problem - persistent storage is implemented.
+Use the steps in this project to customize your deployment:
+* [Customizing your deployment](docs/customising-deployment.md)
 
-Alternative commands and start up [tutorial with AWS support](https://github.com/Alfresco/alfresco-anaxes-shipyard/blob/master/docs/running-a-cluster.md) You will need to adapt some steps for the Community release.
 
-[Tips and tricks](https://github.com/Alfresco/alfresco-anaxes-shipyard/blob/master/docs/tips-and-tricks.md) for working with kubernetes and ACS. Some steps and commands need to be adapted for the Community release.
+## Other information
 
+* See alternative commands and start up [tutorial with AWS support](https://github.com/Alfresco/alfresco-anaxes-shipyard/blob/master/docs/running-a-cluster.md)
+* [Tips and tricks](https://github.com/Alfresco/alfresco-anaxes-shipyard/blob/master/docs/tips-and-tricks.md) for working with Kubernetes and Alfresco Content Services.
 
-#### Notes:
-* You can also change those values when deploying the helm chart by running ```helm install alfresco-incubator/alfresco-content-services --set repository.image.tag="yourTag" --set share.image.tag="yourTag"```.
-* Hint: Run  ```eval $(minikube docker-env)``` to switch to your minikube docker environment on osx.
+**Note:** You'll need to adapt some steps and commands for the Community release.
```

Comment:
This line is occurring twice, see line #30 with similar meaning.

Reply:
I'll remove line #44